docs/teams/mentoring.md

```diff
@@ -1,68 +1,174 @@
-> 4. Notifying their supervisor if they are unable to mentor a team for a significant period of time.
->
-> There are currently five Local Team Coordinators who cover a rough geographic area of the country.  They will be responsible for you as a Mentor and will hopefully be able to arrange social events throughout the year where you can meet other Mentors.
->
-> Think this is something you would like to be involved in?  We would ideally allocate a Mentor to each of the 54 teams so we need as many of you to volunteer as possible…so get involved and help guide a team to success in this year’s competition!
->
-> If you would like to be considered for this role we require the following information:
->
-> Name?
->
-> Location (where you live/work if this is different)?
->
-> How far are you willing to travel?
->
-> Roughly how often could you commit to visiting a school?
->
-> What skills do you possess which would make you a suitable mentor?
->
-> Have you had any previous experience with SR kits/teams e.g. as a volunteer or a participant?
->
-> Please confirm that you are over 18.
-
-
-* Once replies obtained, make ‘Mentor Sign-Up’ spreadsheet with this info and share with LTCs so that they can review and match Mentors with their teams.
-
-* LTCs should:
-    * contact their team leaders to ask them about when, where and how often they meet to be able to match with a potential mentor (might want to produce a Google Form to streamline the information captured)
-    * plot potential mentors on a map and plot the location of your teams
-    * once received the details of the Team meeting times/locations etc. match up geographically close potential mentors to teams
-    * enter the potential TLA by the mentor on the Mentor Sign-Up spreadsheet to identify any clashes.
-    * contact potential Mentor to inform them of their matched team to confirm availability etc. with Team Leader contact info, HIGHLIGHT that mentor should only be interacting with the Team Leader as the responsible adult (i.e over 18) for the team.  DIRECT them to Page 27 of Ops Manual for further info re. being a Mentor.
-    * provide the Mentor with guidance about what the role entails (our [general guidance](../volunteering/mentor-guidance.md) helps with this)
-    * contact Team Leader to inform them of mentor, giving Mentor contact info.
-    * check up on progress of Mentoring relationship through email updates.
+Where possible we try to arrange that competitor teams recieve a regular mentor.
+As well as providing support and guidance to the team, this volunteer typically
+becomes the competitors' main contact with Studdent Robotics.
+
+For this role, mentors need good access to information which may be pertinent to
+competitors and compliment the information we send directly to team leaders.
+
+This page provides an overview of how we organise mentoring and is intended both
+for potential mentors and a team coordinator looking to organise mentoring.
+
+## What mentoring entails
```

Comment:
Do we want to also add something here about the mentor being another person to bounce ideas off? (and one who is perhaps slightly detached from the team's own thinking) Building a rapport with the team is definitely an important part of mentoring, so perhaps.

Reply:
I think we should. It's quite an important detail that a mentor not be "just another team member"

Reply:
9dc0230. Not entirely happy with the framing though as I bet a lot of people might read that and think "I don't know how to do that".